mod-search/src/test/resources/samples/test-data/instances.json

```diff
@@ -361,7 +361,7 @@
           "name": "Brie.",
           "contributorTypeId": "6e09d47d-95e2-4d8a-831b-f777b8ef6d81",
           "contributorTypeText": "",
-          "contributorNameTypeId": "d376e36c-b759-4fed-8502-7130d1eeff39"
+          "contributorNameTypeId": "e8b311a6-3b21-43f2-a269-dd9310cb2d0a"
```

Comment:
Do IDs changes affect browse contributors tests?

Reply:
Very good point.
- Edit browse contributors tests to have more variations of ids
- Attached screen of passed tests.